command/certificate/needsRenewal.go

```diff
@@ -48,11 +52,22 @@ Check if certificate.crt has passed 66 percent of its validity period:
 $ step certificate needs-renewal ./certificate.crt
 '''
 
+Check if any certificate in the bundle has passed 66 percent of it's validity period:
```

Comment:
Clarify in the first example that is the leaf certificate. `Check if the leaf certificate.crt ...`